### ISSUE TITLE
Inherited fields not being pick up by GraphQL plugin

### ISSUE THREAD

**Issue by elie222** [nestjs/graphql#670]
## I'm submitting a...

<pre><code>
[ ] Regression 
[ x ] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

When using inheritance graphql-plugin doesn't pick up `Field` changes. For example:

```ts
@Field(type => Int, { nullable: true })
```

```gql
version: Float
```

Instead of:

```gql
version: Int
```

## Expected behavior

Should emit `Int` instead of `Float`.

## Minimal reproduction of the problem with instructions

```
import { Field, Int, ObjectType } from '@nestjs/graphql'
import { CreateDateColumn, PrimaryGeneratedColumn, UpdateDateColumn, VersionColumn } from 'typeorm'

@ObjectType()
export class SharedEntity {
  @PrimaryGeneratedColumn('uuid')
  id: string

  @Field(type => Int, { nullable: true })
  @VersionColumn({ nullable: true, default: 1 })
  version?: number
}

@ObjectType()
@Entity('school')
export class School extends SharedEntity {
  @Column('varchar', { length: 500, unique: true })
  name: string
}
```

## What is the motivation / use case for changing the behavior?

To reuse code.

## Environment

<pre><code>
Nest version: 
    "@nestjs/core": "^7.0.2",
    "@nestjs/graphql": "^7.0.6",

 
For Tooling issues:
- Node version: v12.7.0
- Platform:  Mac
</code></pre>

**Comment by kamilmysliwiec** [comment]
Please, provide a minimal repository which reproduces your issue.

**Comment by elie222** [comment]
Here's a repro:

https://github.com/elie222/nest-graphql-inheritance-broken

Entity has a `Int` for version:
https://github.com/elie222/nest-graphql-inheritance-broken/blob/master/src/post.entity.ts

Schema has a `Float` for version:
https://github.com/elie222/nest-graphql-inheritance-broken/blob/master/schema.gql

It doesn't need running but if needed:

```
yarn install
yarn start
# http://localhost:4000/graphql
```

**Comment by elie222** [comment]
I used the starter kit and added graphql. Not sure what you mean by minimal

Best,
Eliezer

**Comment by kamilmysliwiec** [comment]
Thanks! Fixed in 7.0.10
